compile/fullCompile/makeCompileArgsFull.m

```diff
@@ -3,8 +3,9 @@
 % Define the arguments for compiling reflectivityCalculation
 % using codegen.
 
-%% Define argument types for entry-point 'reflectivityCalculation'.
+%% Define argument types for entry-point 'RATMain'.
 maxArraySize = 10000;
+maxDataSize = 10000;
```

Comment:
I don't think these bounds are needed. Theses values should just be `Inf `except where we actually have a bound, to the best of my knowledge the memory is allocated as needed so not sure the reason for the artificial handicap. @arwelHughes do you remember the original reason for the max size?  

Is it possible that something like the data (`problemCells{2} = inputStruct.data`) can exceed our artificial limit of 10000 rows maybe someone doing a simulation or something?